### DISCUSSION TITLE
Automatiser l'ajout de factures par ROC sur Mes Achats

### DISCUSSION THREAD

**Discussion by ghost** [betagouv/ma-cantine#1190] [Idées]
La suggestion d'une utilisatrice (M. G.), que je trouve excellente : automatiser l'ajout de factures sur Mes Achats par reconnaissance optique de caractères. Le gestionnaire n'aurait qu'à téléverser le scan ou la photo de sa facture sur Mes Achats, le site Ma Cantine créerait ensuite automatiquement la ou les lignes correspondant à la facture.

Intérêt : immense gain de temps pour l'utilisateur.

Difficulté : à mon avis, grande complexité technologique pour mettre en place cette fonctionnalité => projet de long terme

**Comment by JenniferStephan** [comment]
Hello @YodBetYod, c'est bien prévu sur du long-terme; on aimerait bien d'associer à une startp privé qui a déjà la technique ;)

**Reply by ghost** [reply]
@JenniferStephan, voici 2 boîtes que m'a recommandées mon frère dans le domaine de la ROC :
- Expensify : appli web et mobile, côtée en bourse (la boîte de mon frère l'utilise pour les tickets de caisse des déjeuners des employés)
- Hypatos.ai (le fonds dans lequel travaille mon frère a investi dans cette boîte)
